controllers/api/post-routes.js

```diff
@@ -26,7 +26,10 @@ router.get('/', (req, res) => {
       'locations',
       'activities',
       'cuisine',
-      'metric'
+
```

Comment:
we need 'codes' (instead of 'metric') added here, but can fix in later merge

Reply:
Sounds good. I got confused when Christine's merge went though, I think this was an older develop branch.